cypress/e2e/hub/remotes.cy.ts

```diff
@@ -86,16 +117,29 @@ describe('Remotes', () => {
     cy.get('[data-cy="download-concurrency"]').type(Remotes.downloadConcurrency);
     cy.get('[data-cy="rate-limit"]').type(Remotes.rateLimit);
     cy.get('[data-cy="tls_validation"]').click();
+    cy.get('[data-cy="requirements_file"]')
+      .click()
+      .focused()
+      .type('{ctrl}a')
```

Comment:
this is probably platform-dependent, might not work on a mac.. would `{home}` be better?

Reply:
@himdel updated.